jina/resources/executors.requests.Match2DocRanker.yml

```diff
@@ -0,0 +1,8 @@
+on:
+  ControlRequest:
+    - !ControlReqDriver {}
+  SearchRequest:
+    with:
+      traversal_paths: [ 'r' ]
```

Comment:
Do we need this? This is the same as the default value.

Reply:
Done!